.github/workflows/check-http-status-code.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
I've added the `set -e` in the new scripts and decided to add it here too. In my opinion, if you want all the CI flow to execute correctly in your PR you prefer it to fail as quickly as possible instead of the more detailed way. Anyway, adding it made the script fail for the errors that you found that I have missed.

I'm not sure that I understood correctly what you meant in line 13.

Reply:
I meant that that IF statement I quoted is unreachable code, it will never execute because `$?` will always be `0` thanks to `set -e`